docs/functions/functions.json

```diff
@@ -11,12 +11,7 @@
 				}
 			],
 			"id": "+",
-			"unary": false,
-			"shown": [
```

Comment:
Yikes, why was this saved? Did we forget to specify `mv-storage="none"` to that property?

Reply:
It seems to me, in early versions of Mavo computed properties might have been stored. And those are leftovers from those times. This property is not stored anymore.